src/assets/images/pencil-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="49" height="49" viewBox="0 0 49 49" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What do you feel about wrapping all of our svgs in a react component? That way we can drive props into the svgs and change/animate changing colours & size

Reply:
hmm yeah we can do that 👍